DRAFT.md

```diff
@@ -40,6 +40,80 @@ This proposal is a fusion of two earlier individual proposals, P1950 and P0201.
 The design of the two proposed class templates is sufficiently similar that they
 should not be considered in isolation.
 
+## History
+
+### Changes from R2
+
+* Add explicit to constructors.
+
+* Add constructor indirect(U&& u, Us&&... us) overload and requisite constraints.
+
+* Add constructor polymorphic(allocator_arg_t, const Allocator& alloc) overload.
+
+* Add discussion on similarities and dissimilarities with variant.
+
+* Add table of breaking and non-breaking change to appendix C.
+
+* Add missing comparison operators and ensure they are all conditionally noexcept.
+
+* Add argument deduction guides for `std::inmdirect`.
+
+* Additions to acknowledgements.
+
+* Address wording for `swap()` relating to `noexcept`.
+
+* Address constraints wording for `std::indirect` comparison operators.
+
+* Copy constructor now uses allocator_traits select_on_container_copy_construction.
+
+* Ensure swap and assign with self are nops.
+
+* Move feature test macros to [version.syn].
+
+* Remove `std::optional` specializations.
```

Comment:
I think we did this in an earlier revision, happy to be wrong.

Reply:
The change we did this in was: https://github.com/jbcoe/value_types/commit/7fec8a37a74ff50bc0456041402565a6296a942e

This was on November 11th (the Saturday we made many changes in the hotel lobby), we had previously presented R2 of the paper at the end of Friday the afternoon before. So, I'm pretty sure its this revision unless I've missed something else?

Reply:
My mistake. Thanks for double checking.